config/database.yml

```diff
@@ -56,7 +58,9 @@ development:
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
-  <<: *default
+  adapter: sqlite3
```

Comment:
Need to clarify with @imuchnik, but we might need to leave this as default for tests to run correctly.

@coreyd303 did you run the test suite to verify that everything still runs smoothly?